src/lib/command-helpers.mjs

```diff
@@ -14,10 +16,11 @@ export async function getSimpleClient(argv) {
     const faunadb = (await import("faunadb")).default;
     const { Client, query: q } = faunadb;
     const { hostname, port, protocol } = new URL(argv.url);
+    const scheme =  protocol?.replace(/:$/, ""),
     client = new Client({
       domain: hostname,
-      port,
-      scheme: protocol?.replace(/:$/, ""),
+      port: Number(port),
+      scheme: /** @type {('http'|'https')} */ (scheme),
```

Comment:
is this the best way to type it? Can we type the `Client`

Reply:
The v4 driver client has the type `'http' | 'https' | undefined`, which seems pretty accurate to me. I don't think we want to loosen that type.

Reply:
oh was mostly referring to the syntax/inline comment way of doing things